markdown/appendices/resources.md

```diff
@@ -25,10 +25,25 @@ title: Other Doomsday Resources
 
 - [Legacy Breakfast](https://legacy-breakfast.com/tag/doomsday/)
 - [Eternal Central](https://www.eternalcentral.com/tag/doomsday/)
+- [The Dead Format episode 106 ft. Nevilshute](https://soundcloud.com/the-dead-format/episode-106-doomsday-with-nevilshute-martin-nielsen)
 
 ## Videos
 
+- [DK_TTGL runs a League with Grixis Meandeck (June 2020) - Portugese]
+  (https://www.youtube.com/watch?v=eMgfFlx0_10)
+- [Jarvis Yu ft. Ari Lax do a League with 4C Meandeck (July 2020)]
+  (https://www.youtube.com/watch?v=mn4GYFoVrRA)
 - [Anuraag Das's VOD featuring monkeyscantcry (march 2020)]
   (https://www.twitch.tv/videos/565641017)
 - [d8dk32's top 8 at the Legacy Titanium Series 1K Challenge (august 2017)]
   (https://www.youtube.com/watch?v=VmmR_3pG0Rs&t=38m3s)
+
+## Highlight Reel
```

Comment:
@Doishy I think the Twitch clips are deleted after some time (months). Can we upload these somewhere else ie. authors' YouTube perhaps?

Reply:
Possibly?
Not sure how we could go about that.